Source/JavaScriptCore/disassembler/zydis/Zydis/ZycoreList.c

```diff
@@ -94,7 +94,7 @@ static ZyanStatus ZyanListAllocateNode(ZyanList* list, ZyanListNode** node)
                 return ZYAN_STATUS_INSUFFICIENT_BUFFER_SIZE;
             }
 
-            *node = (ZyanListNode*)((ZyanU8*)list->buffer + size);
+            memcpy(*node, (ZyanU8*)list->buffer + size, sizeof(ZyanListNode));
```

Comment:
This is not correct. The memcpy code is copying ZyanListNode, but original code was copying a pointer. So it is wrong.
I think this warning is basically useless. Right fix is suppressing this warning.